src/test/scala/TypeCheckerSpec.scala

```diff
@@ -1003,4 +1003,16 @@ class TypeCheckerSpec extends FunSpec {
       }
     }
   }
+
+  describe("Imports") {
+    it("adds externs into type checking scope") {
+      typeCheck("""
+        import "print.cpp" {
```

Comment:
I know these are just the tests, but `#include`ing *.cpp files is rarely what you want—maybe use `print.h` instead.

Reply:
Fixed.